app/src/main/java/http_server/ResponseHandler.java

```diff
@@ -3,37 +3,41 @@
 public class ResponseHandler {
```

Comment:
Mentioned this in the other comment already but we may want to consider renaming this since this class seems to be the one "building" a response.

Reply:
Should I go ahead with this refactor or make this a later ticket as you mentioned we previously talked about?